api/v1/observer/observer.proto

```diff
@@ -71,7 +71,10 @@ message GetFlowsRequest {
     // Number of flows that should be returned. Incompatible with `since/until`.
     uint64 number = 1;
 
-    // Unused 2
+    // Dump all flows. Incompatible with `since/until`, `follow` and any filter.
+    // It also does not guarantee chronological ordering of flows. This is
```

Comment:
this says it does not guarantee, but the comment in the other place says they are printed in chronological order

Reply:
I initially thought ordering flows could be a problem and it turns out it's not. Will remove this sentence.